Maths/TwinPrime.js

```diff
@@ -0,0 +1,43 @@
+function isPrime (n) {
```

Comment:
We already have prime checking algorithms, use those instead of duplicating code.